lib/src/main/java/com/knuddels/jtokkit/Encodings.java

```diff
@@ -10,12 +10,21 @@ public final class Encodings {
 	 *
 	 * @return the new {@link EncodingRegistry}
 	 */
-	public static EncodingRegistry newDefaultEncodingRegistry() {
-		final DefaultEncodingRegistry registry = new DefaultEncodingRegistry();
+	public static EncodingRegistry newEagerEncodingRegistry() {
```

Comment:
I think we should keep the `newDefaultEncodingRegistry` naming here in the API. While your naming is more expressive, it will break every consumer of this library on upgrade, which I would like to avoid

Reply:
Ah, I forgot that this is a library. I'll change the name back to its original one.

Reply:
Thank you! :slightly_smiling_face: